python/cudf/cudf/_lib/round.pyx

```diff
@@ -31,19 +25,15 @@ def round(Column input_col, int decimal_places=0, how="half_even"):
     if how not in {"half_even", "half_up"}:
         raise ValueError("'how' must be either 'half_even' or 'half_up'")
 
-    cdef column_view input_col_view = input_col.view()
-    cdef unique_ptr[column] c_result
-    cdef cpp_rounding_method c_how = (
-        cpp_rounding_method.HALF_EVEN if how == "half_even"
-        else cpp_rounding_method.HALF_UP
+    cdef rounding_method c_how = (
```

Comment:
Remove the cdef here so that you don't have to cimport rounding_method above. Everything should still work as expected.